.github/workflows/_deploy-hl7-notification-cdk.yml

```
@@ -104,18 +104,12 @@ jobs:
-                                            --workspace=packages/mllp-server \
-                                            --workspace=packages/infra
+          npm ci --ignore-scripts --no-fund
         working-directory: "metriport/"
 
       - name: Build relevant packages
         run: |
-          npm run build:cloud --workspace=packages/shared
-          npm run build:cloud --workspace=packages/core
-          npm run build:cloud --workspace=packages/mllp-server
-          npm run build:cloud --workspace=packages/infra
+          npm run build:cloud 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)